events/wc_massacre_shadow_council.txt

```diff
@@ -0,0 +1,479 @@
+namespace = WCMSC
+
+#Becoming Warchief
+narrative_event = {
+	id = WCMSC.1
```

Comment:
Zuzu comment: 

>WCMSC.1 is too laggy
It tries to fire everyday
For every character
Look at how it's done in vanilla
For example, when e_spain is created
It finishes reconquista
Event ID is HF.49451
Look where vanilla fires it